code/__HELPERS/game.dm

```diff
@@ -13,10 +13,10 @@
 
 
 /// Checks all conditions if a spot is valid for construction , will return TRUE
-/proc/IsValidForResinStructure(turf/target, needsSupport = FALSE, mob/builder)
+/proc/is_valid_for_resin_structure(turf/target, needs_support = FALSE, mob/builder)
 
 	if(!target || !istype(target))
-		return FALSE
+		return TRUE
```

Comment:
should return an ERROR or NO_ERROR, not true or false. True or false are considerered boolean in our codebase